### PR TITLE
Add Project Access Token related endpoints

### PR DESCRIPTION
**Pull request by balasankarc**
Add the endpoints mentioned in https://docs.gitlab.com/ee/api/project_access_tokens.html

### CODE REVIEW

File: lib/gitlab/client/projects.rb [hunk 1]
```diff
@@ -715,5 +715,74 @@ def project_deploy_tokens(project, options = {})
...
+    # Get a specific project access token.
+    #
+    # @example
+    #   Gitlab.project_access_token(42, 1234)
+    #
+    # @param [Integer, String] project The ID or path of a project.
+    # @param [Integer] token_id The ID of the project access token.
+    #
+    # @return [Gitlab::ObjectifiedHash] Information about the specified project access token.
+    def project_access_token(project, token_id, options = {})
```
Comment (NARKOZ) [inline lib/gitlab/client/projects.rb:741]: Why do we need the `options` argument here?

Comment (balasankarc) [reply lib/gitlab/client/projects.rb]: @NARKOZ You're right. That is an oversight. Fixed.

**Review by NARKOZ — APPROVED** [review]

**Comment by NARKOZ** [comment]
Thank you ❤️

**Comment by balasankarc** [comment]
@NARKOZ Would it be possible to have 5.1.0 cut with the recent changes? Thanks.

**Comment by NARKOZ** [comment]
Sorry for the wait. Version 5.1.0 has just been released. Please see https://github.com/NARKOZ/gitlab/releases

**Pull request merged**